ui-participant/src/PageNotFound.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCircleExclamation } from '@fortawesome/free-solid-svg-icons'
+import { usePortalEnv } from './providers/PortalProvider'
+import Navbar from './Navbar'
+import { Link } from 'react-router-dom'
+
+/**
+ * Displays when there is an unmatched participant route.
+ */
+export default function PageNotFound() {
+  const portalEnv = usePortalEnv()
+  const supportEmail = portalEnv.portalEnv.portalEnvironmentConfig.emailSourceAddress
+  return (
+    <div className="container-fluid bg-white min-vh-100 d-flex flex-column p-0">
+      <Navbar aria-label="Primary"/>
+      <main className="flex-grow-1 p-2 d-flex flex-column justify-content-center">
+        <div className="fs-1 fw-bold d-flex justify-content-center">
+          <div>
+            <FontAwesomeIcon className="me-2" icon={faCircleExclamation}/>
+            <span>Page not found</span>
+          </div>
+        </div>
+        <div className="fs-2 fw-light d-flex justify-content-center text-center">
+          <div>
+            <span>
+              The page you requested was not found.
+
+              If the problem persists, please
+              contact <a href={`mailto:${supportEmail}`}>{supportEmail}</a>.
```

Comment:
```suggestion
              contact <a href={`mailto:${portalEnv.portalEnvironmentConfig.emailSourceAddress}`}>{supportEmail.portalEnvironmentConfig.emailSourceAddress}</a>.
```

Reply:
It's already using the `emailSourceAddress`, it's just pulled out into a const above. I'll rename the const to `studyStaffEmail` or something a little more clear

Reply:
ah, sorry I missed that -- just saw the screenshot and assumed.  We should probably change the demo emailSourceAddress to something like support@heartdemo.org so it's easier to see that

Reply:
ah, I forgot, the email has to be a real one since it will be the source email for sendgrid emails from the demo.  hmm...